code/modules/wiremod/shell/structure/bot.dm

```diff
@@ -18,7 +18,7 @@
 		/datum/component/shell, \
 		unremovable_circuit_components = list(new /obj/item/circuit_component/bot), \
 		capacity = SHELL_CAPACITY_LARGE, \
-		shell_flags = SHELL_FLAG_USB_PORT, \
+		shell_flags = SHELL_FLAG_USB_PORT|SHELL_FLAG_REQUIRE_ANCHOR, \
```

Comment:
This is somewhat inconsistent.
Bot shells shouldn't need to be anchored to be functional. They should work without being anchored.

Reply:
Ah, I did consider that, but bot shells are noted as immobile both in the printer design description:
```
(file: code/modules/research/designs/wiremod_designs.dm)
510  /datum/design/bot_shell
511  	name = "Bot Shell"
512  	desc = "An immobile shell that can store more components. Has a USB port to be able to connect to computers and machines."
```
AND in the documentation in `bot.dm`:
```
(file: code/modules/wiremod/shell/bot.dm)
1 /**
2  * # Bot
3  *
4  * Immobile (but not dense) shells that can interact with world.
5  */
```
Hence the choice.
Why do you think it's inconsistent?

Regardless of whether it's inconsistent, I do think it *is* a good idea to separate those flags at the very least for posterity.

Reply:
Probably misattributed.
Bot was the first shell that was made when I was creating integrated circuits but they were never intended to be forced into being anchored - that's why the server shell exists. I'd rather you adjust the description rather than change the functionality.

Reply:
I see. That feels like an odd choice of words given not moving is what immobile means, but fair enough, and thanks for the clarification.
I was originally just going to make them anchorable to be consistent to all the other structure shells, and honestly thought it felt off to make them require it too, before seeing the code documentation and thinking "...so it probably _should_ be that, huh."
I'll look into it together with the other suggestions when I have the time.